cfme/services/myservice/ui.py

```diff
@@ -354,6 +354,7 @@ def download_file(self, extension):
 def reconfigure_service(self):
     # TODO refactor this method - it does nothing at the moment. Bug 1575935
```

Comment:
@jaryn does this comment still apply?

Reply:
I think it still applies. The `reconfigure_service` is not changing anything in the service. It basically just checks we are able to save it as it is.